pkg/setting/setting_plugins.go

```diff
@@ -34,13 +36,61 @@ func extractPluginSettings(sections []*ini.Section) PluginSettings {
 var (
 	defaultPreinstallPlugins = map[string]InstallPlugin{
 		// Default preinstalled plugins
-		"grafana-lokiexplore-app":      {"grafana-lokiexplore-app", "", ""},
-		"grafana-pyroscope-app":        {"grafana-pyroscope-app", "", ""},
-		"grafana-exploretraces-app":    {"grafana-exploretraces-app", "", ""},
-		"grafana-metricsdrilldown-app": {"grafana-metricsdrilldown-app", "", ""},
+		"grafana-lokiexplore-app":      {ID: "grafana-lokiexplore-app"},
+		"grafana-pyroscope-app":        {ID: "grafana-pyroscope-app"},
+		"grafana-exploretraces-app":    {ID: "grafana-exploretraces-app"},
+		"grafana-metricsdrilldown-app": {ID: "grafana-metricsdrilldown-app"},
 	}
 )
 
+func (cfg *Cfg) migrateInstallPluginsToPreinstallPluginsSync(rawInstallPlugins, installPluginsForce string, preinstallPluginsSync map[string]InstallPlugin) {
+	if strings.ToLower(installPluginsForce) == "true" || rawInstallPlugins == "" {
```

Comment:
nit: we could also add a debug line here, to say that we are skipping this process